lancetipton/parkin
# V5-disabled tags and timeouts

## Updates

* Update deps
     * Switch to jasmine-2 test runner
     * Need to investigate migrating at somepoint
* Add step to step context object
     * Add filtering background from tags in runner
* Add global timeout to test 
  * Move step timeout to test where it makes more sense
  * Now can work long side other test frameworks that take `timeout` as the 3rd argument
    * i.e. playwright / jest   
* Add tags to steps and background
* Change parkin.run options
      * Allow passing tags to filter and disable
      * Will also allow future updates to tags
      * Add disabled tag check for feature and properties


BREAKING CHANGE: Changes the parkin.run options
